Gem/Code/Source/BakedShaderVariantExampleComponent.cpp

```diff
@@ -75,7 +75,7 @@ namespace AtomSampleViewer
 
         Data::AssetId modelAssetId;
         Data::AssetCatalogRequestBus::BroadcastResult(
-            modelAssetId, &Data::AssetCatalogRequestBus::Events::GetAssetIdByPath, Products::ModelFilePath, nullptr, false);
+            modelAssetId, &Data::AssetCatalogRequestBus::Events::GetAssetIdByPath, Products::ModelFilePath, AZ::Uuid::CreateNull(), false);
```

Comment:
FYI, brace initialization can be used here `{}` .
The AZ::Uuid class now default constructs to a Null Uuid.

Reply:
Personally I prefer CreateNull vs {}. With brace initialization I need to go look up what UUID does in that case, CreateNull tells me explicitly.

Reply:
If you prefer that, I can do it.  Saying `CreateNull()` seems to be a bit more "explicit" telling users what's being passed.

Reply:
It doesn't matter to be me how one does it.
But all these initializations are now equivalent
```
{}
AZ::Uuid()
AZ::Uuid{}
AZ::CreateNull()
```